tool/tsh/kube.go

```diff
@@ -668,6 +670,8 @@ func newKubeLSCommand(parent *kingpin.CmdClause) *kubeLSCommand {
 	c.Flag("format", defaults.FormatFlagDescription(defaults.DefaultFormats...)).Short('f').Default(teleport.Text).EnumVar(&c.format, defaults.DefaultFormats...)
 	c.Flag("all", "List kubernetes clusters from all clusters and proxies.").Short('R').BoolVar(&c.listAll)
 	c.Arg("labels", labelHelp).StringVar(&c.labels)
+	c.Flag("verbose", "Print full list of labels.").Short('v').BoolVar(&c.verbose)
```

Comment:
Why `verbose` prints all labels? Should that option be called more like `--all` or `--all-labels`?

Reply:
I followed the same criteria used for nodes, databases and apps:
https://github.com/gravitational/teleport/blob/7208411f8fbc3794397040312d06ac4458c6c24b/tool/tsh/tsh.go#L2127-L2143